tools/scripts-core/scripts/extends.js

```diff
@@ -1,8 +1,10 @@
-const fs = require('fs');
-const path = require('path');
-const template = require('lodash.template');
-const { getPreset } = require('../utils/preset');
-const { getUserConfigFile } = require('../utils/env');
+/* eslint-disable no-console */
```

Comment:
Yes documentation of each scripts :sweat: 
No, it is just if you want to extends existing configuration to override.
I bet with the removal of preset it will be more useful (may be even used)